src/QirRuntime/build.py

```diff
@@ -0,0 +1,95 @@
+import sys, os, platform, subprocess, datetime
```

Comment:
Is this still needed? Looks like the logic has been incorporated to `build.ps1`

Reply:
I'd like to keep build.py/test.py for the convenience of local development. If maintaining both these and powershell scripts scripts proves to be a burden we can deprecate.

Reply:
I'd like to keep build/test python scripts for the convenience of the local development. If maintaining both python and powershell scripts proves to be a burden in the future we can deprecate the former.